kb(Button): add top and bottom icon position (#2017)
telerik/blazor-docs: knowledge-base/button-change-icons-position.md

## Description

This KB article answers the following questions:

* How can I show an icon in a Telerik UI for Blazor Button on the right side of the text instead of on the left side?
* How can I show an icon in a Telerik UI Blazor Button on the top and bottom side of the text?
* How can I also include more than one icon in the Button?

## Solution

> You can change the position of the [icons]({%slug common-features-icons%}) and add more icons in the buttons by the following approaches:

Reviewer: ```suggestion
You can change the [icon]({%slug common-features-icons%}) position and add more icons in the Button by using the following approaches:
```

* Nesting the `TelerikIcon` in the `TelerikButton` content.

> * Use CSS to position an icon either above or below the text content.

Reviewer 2: ```suggestion
* To control the position of the icon in relation to the text content of the Button, use CSS.
```

>caption Reposition the Button icon and add more than one icon to the component

![Telerik UI for Blazor Button with two icons](images/button-change-icon-position-example.png)
![Telerik UI for Blazor Button with  icon on top](images/button-change-icon-position-top-example.png)

> ![Telerik UI for Blazor Button with  icon on bottom](images/button-change-icon-position-bottom-example.png)

Reviewer: We try to remove screenshots, rather than add new ones. Normally, we only leave screenshots that are animated and show the component behavior as a process.

Reviewer 2: It seems that there are two spaces between "with" and "icon".

````CSHTML
@*This button has two icons, one on the left and one on the right side of the text*@

> <TelerikButton Class="my-button">

Reviewer: Use a meaningful CSS class name, for example `two-icons` or `icons-top-bottom`.

    <TelerikSvgIcon Icon="@SvgIcon.InfoCircle" />
    Info
    <TelerikSvgIcon Icon="@SvgIcon.InfoCircle" />
</TelerikButton>

> @* This button has an icon on top of the text. *@

Reviewer 2: ```suggestion
@* This button has an icon above the text. *@
```
